frontend/src/Context/PreviewPublisherProvider/usePreviewPublisher/usePreviewPublisher.tsx

```diff
@@ -164,13 +164,14 @@ const usePreviewPublisher = (): PreviewPublisherContextType => {
    */
   const handleAccessDenied = useCallback(
     async (event: AccessDeniedEvent) => {
-      const deviceDeniedAccess = event.message?.startsWith('Microphone') ? 'microphone' : 'camera';
+      const deviceDeniedAccess = (
+        event.message?.startsWith('Microphone') ? 'microphone' : 'camera'
+      ) as PermissionName;
```

Comment:
Note: the `as` casting is needed due to otherwise getting a TypeScript error: `Type '"camera" | "microphone"' is not assignable to type 'PermissionName'.
  Type '"camera"' is not assignable to type 'PermissionName'.` 

`PermissionName` looks like the following: `type PermissionName = "camera" | "geolocation" | "microphone" | "midi" | "notifications" | "persistent-storage" | "push" | "screen-wake-lock" | "storage-access"`